helpers/db_indexes.go

```diff
@@ -0,0 +1,20 @@
+package helpers
```

Comment:
We probably don't need an entire new file for just this helper. Can refactor later. I'd say move to helpers.go.